worlds/sc2/docs/setup_fr.md

```diff
@@ -0,0 +1,214 @@
+
+Pour un monde *StarCraft 2* seule, cette option ne fait rien. 
+Il s'agit d'une option d'Archipelago qui permet de balancer la progression d'un monde relativement aux autres mondes en 
+interchangeant les *items* de progression dans les *sphères*.
+Si le *Progression Balancing* d'un monde est plus grand que ceux des autres, les *items* de progression de ce monde ont 
+plus de chance d'être obtenus tôt et vice-versa si sa valeur est plus petite que celle des autres mondes.
+Cependant, *StarCraft 2* est beaucoup plus permissif en termes d'*items* qui sont nécessaires pour progresser.
+Pour cette raison, cet ajustement a souvent peu d'influence sur la capacité de progresser dans un monde de 
+*StarCraft 2*. 
+De plus, cette option augmente le temps de génération d'un *MultiWorld*, alors certains recommandent de le définir à 
+zéro pour *StartCraft 2*.
+
+#### Comment est-ce que je définis une liste d'*items*, e.g. pour l'option *excluded items*?
+
+Vous pouvez lire sur la syntaxe des conteneurs dans le format *yaml* à la page 
+[*YAML specification*](https://yaml.org/spec/1.2.2/#21-collections). 
+Pour les listes, chaque *item* doit être sur sa propre ligne et doit être précédé par un trait d'union.
+
+```yaml
+excluded_items:
+  - Battlecruiser
+  - Drop-Pods (Kerrigan Tier 7)
+```
+
+Une liste vide est représentée par une paire de crochets: `[]`. 
+Il s'agit de la valeur par défaut dans le modèle de base, ce qui devrait vous aider à apprendre à utiliser cette 
+syntaxe.
+
+#### Comment est-ce que je fais pour avoir des *items* dès le départ?
+
+L'option *starting inventory* est un *map* et non une liste. 
+Ainsi, elle permet de spécifier le nombre de chaque *item* avec lequel vous allez commencer.
+Sa syntaxe consiste à indiquer le nom de l'*item*, suivi par un deux-points, puis par un espace et enfin par le nombre 
+désiré de cet *item*.
+
+```yaml
+start_inventory:
+  Micro-Filtering: 1
+  Additional Starting Vespene: 5
+```
+
+Un *map* vide est représenté par une paire d'accolades: `{}`. 
+Il s'agit de la valeur par défaut dans le modèle de base, ce qui devrait vous aider à apprendre à utiliser cette 
+syntaxe.
+
+#### Comment est-ce que je fais pour connaître le nom des *items* et des *locations* dans *StarCraft 2 Archipelago*? 
+
+La page [*datapackage*](/datapackage) d'Archipelago liste l'ensemble des *items* et des *locations* de tous les jeux 
+que le site web prend en charge actuellement, dont ceux de *StarCraft 2*.
+
+Vous trouverez aussi la liste complète des *items* de *StarCraft 2 Archipelago* à la page 
+[*Icon Repository*](https://matthewmarinets.github.io/ap_sc2_icons/). 
+Notez que cette page contient diverses informations supplémentaires sur chacun des *items*.
+Cependant, l'information présente dans cette dernière peut différer de celle du *datapackage* d'Archipelago puisqu'elle 
+est générée, habituellement, à partir des développements récents de *StarCraft 2 Archipelago* qui n'ont peut-être pas 
```

Comment:
"Développements récents" can be understood as "recent news" instead of "latest updates". Instead, it should be "à partir de la dernière version de *Starcraft 2 Archipelago*".

Reply:
"développements récents" was meant as the dev branch. I feel that " dernière version de Starcraft 2 Archipelago" would mean the lastest version merged to Archipelago, which is not what it does. With that context, is that alright?

Reply:
I guess the best way to say it would be to explicitely say that the version merged in Archipelago might not be the last version of the randomizer instead. So, instead something like "Cependant, il est possible que le site web d'Archipelago utilise une précédente version de *Starcraft 2 Archipelago*, auquel cas l'information présente sur cette image peut différer de celle de votre monde", or something like that ? Or we could call it a beta version, or a bleeding edge version, or something like that.

Reply:
A modification, agreed in private messages, was applied.